commerce-inventory-api/bnd.bnd

```diff
@@ -2,7 +2,7 @@ Bundle-Name: Liferay Commerce Inventory API
 Bundle-SymbolicName: com.liferay.commerce.inventory.api
 Bundle-Version: 1.0.0
 Export-Package:\
-com.liferay.commerce.inventory.constants,\
+com.liferay.commerce.inventory.constants
```

Comment:
@hhuijser is this right?

Reply:
@ethanbustad 
Nope, that's not right. Good catch!
Will look into that one.
Shall I resend in a few separate pulls?